app/helpers/duplicate_logs_helper.rb

```diff
@@ -22,4 +22,9 @@ def duplicate_logs_continue_button(all_duplicates, duplicate_log, original_log_i
   def duplicate_logs_action_href(log, page_id, original_log_id)
     send("#{log.model_name.param_key}_#{page_id}_path", log, referrer: "interruption_screen", original_log_id:)
   end
+
+  def change_duplicate_logs_action_href(log, page_id, all_duplicates, original_log_id)
+    remaining_duplicate_id = all_duplicates.map(&:id).reject { |id| id == log.id }.first
```

Comment:
would renaming this to `first_remaining_duplicate_id` be clearer if there can be multiple remaining at this stage?

Reply:
sure, I think that's a good idea!